src/app/marketplace/marketplace-analytics-dimesions-metrics-explorer/marketplace-analytics-dimensions-metrics-explorer.component.ts

```diff
@@ -0,0 +1,159 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import analyticsConcepts from './marketplace-analytics-dimensions-metrics-concepts';
+import analyticsCubes from './marketplace-analytics-dimensions-metrics-cubes';
+import { map, pluck } from 'rxjs/operators';
+import { HttpClient } from '@angular/common/http';
+import { animate, state, style, transition, trigger } from '@angular/animations';
+import { MatCheckboxChange } from '@angular/material';
+import { Subscription } from 'rxjs';
+
+interface MatadataColumnAttributes {
+  type: string;
+  dataType: string;
+  group: string;
+  status: string;
+  uiName: string;
+  description: string;
+  allowedInSegments: string;
+  addedInApiVersion: string;
+}
+
+interface MetadataColumn {
+  id: string;
+  kind: string;
+  attributes: MatadataColumnAttributes;
+}
+
+export interface DimensionsMetricsSelection {
+  dimensions: string[];
+  metrics: string[];
+}
+
+@Component({
+  selector: 'app-marketplace-analytics-dimensions-metrics-explorer',
+  templateUrl: './marketplace-analytics-dimensions-metrics-explorer.component.html',
+  styleUrls: [ './marketplace-analytics-dimensions-metrics-explorer.component.scss' ],
+  animations: [
```

Comment:
since this is second time we use the same animation we may put it in a `animations.ts` file and import it here.